flow/probes/ebpf.go

```diff
@@ -271,21 +293,31 @@ func (p *EBPFProbe) run() {
 					ebpfFlow = &EBPFFlow{
 						start: now,
 						last:  start.Add(time.Duration(lastK - startKTimeNs)),
+						Kflow: kernFlow,
 					}
 					ebpfFlows[kernFlow.key] = ebpfFlow
+
 				}
+                                if ok{
+                                    updateFromEBPF(ebpfFlow,&kernFlow)
+				    kernFlow=ebpfFlow.Kflow
```

Comment:
I think you can remove this line and you can change the flowFromEBPF definition to remove the kernFlow parameter.

Reply:
Done